src/App.css

```diff
@@ -20,3 +20,92 @@
 .App-link {
   color: #09d3ac;
 }
+
+.main-container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  background-color: #012a46;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  position: relative;
+  top:25%
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+form input {
+  border: none;
+  margin: auto;
+  padding: 20px;
+  margin-bottom: 19px;
+  border-radius: 30px;
+  width: 205px;
+  background: rgba(229, 229, 229, 0.15);
+  font-family: fira code;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 24px;
+  color:#FF7661;
+}
+
+.startscreen {
+  display: flex;
+  flex-direction: column;
+  width: 30%;
+}
+
+ .go {
+  border: none;
+  margin: auto;
+  padding: 18px;
+  margin-top: 12px;
+  border-radius: 30px;
+  width: 180px;
+  background: rgba(229, 229, 229, 0.15);
+  color: #FF7661;
+  font-weight: bold;
+  font-family: fira code;
+  font-size: 24px;
+}
+
+::placeholder{
+  color:rgba(255, 118, 97, 0.55);
+}
+
+.button-one {
+  border: none;
```

Comment:
duplication between this and .go and button2
you should factor out the commonalities!